### DISCUSSION TITLE
Hi thank you for the update

### DISCUSSION THREAD

**Discussion by Narugakuruga** [TryTwo/dolphin#7] [General]
I'm a dolphin user and I used to find out dolphin was not emulating Monster Hunter Tri correctly.(Demonstration in [this article](https://cn.dolphin-emu.org/blog/2020/10/05/dolphin-progress-report-july-and-august-2020/#fix-monster-hunter-tri-bloom-disabled-patch))
And I searched for Google and found your build and solution. I compile the code with Bloom strength and radius setting in 2022 and it was the most significant improvement to the game. 

And thank you for updating to the latest branch of dolphin with radius setting. It's the love and devotion to keep working on something!

**Comment by TryTwo** [comment]
It's a great improvement for some games. It's unfortunate I haven't been able to get it fully implemented in normal dolphin builds.  Some day though...

**Comment by Narugakuruga** [comment]
I found a glitch where some textures is displayed as green color in monster hunter tri. Is it possible to be related to shader changes?

**Comment by TryTwo** [comment]
Thanks!  It was a problem with having to reorganize some code due to tweaks from other devs. Textures marked with scaled_by_half were checked too late. Reuploaded fixed version

**Comment by TryTwo** [comment]
Uploaded a experimental version, give it a try for me!

**Comment by Narugakuruga** [comment]
I've just received the update and test it for thirty-ish minutes. So far every post process feature seems to functioning. The texture glitch is fixed also !

**Comment by Narugakuruga** [comment]
Also played with the new gamma correction setting. Set gamma value to around 2.20-2.30
Many games looks better than raw output 
after the compensation. Which is also a good addon to the bloom effect.

**Comment by Narugakuruga** [comment]
I also noticed the bloom around the corner is processed with noticably better appearance. The center area of the bloom behaves consistently as the behavior before.

**Comment by TryTwo** [comment]
Thanks for testing
